xbmc/cores/AudioEngine/Sinks/AESinkPULSE.cpp

```diff
@@ -385,6 +396,20 @@ static CAEChannelInfo PAChannelToAEChannelMap(const pa_channel_map& channels)
   return info;
 }
 
+static void ModuleInfoCallback(pa_context* c, const pa_module_info *i, int eol, void *userdata)
```

Comment:
wrap this function inside `#if PA_CHECK_VERSION(10,0,0)` (unused function warning)